pkg/minikube/cluster/cluster.go

```diff
@@ -437,7 +440,13 @@ func GetVMHostIP(host *host.Host) (net.IP, error) {
 		}
 		return ip, nil
 	case "virtualbox":
-		ip, err := getIPForInterface("vboxnet0")
+		out, err := exec.Command("VBoxManage", "showvminfo", "minikube", "--machinereadable").Output()
```

Comment:
Do we need docker machine's OS-specific implementation for windows?  On the rest of the platforms it just finds it in the path, but on windows it looks specifically in a few places too.

https://github.com/docker/machine/blob/e1a03348ad83d8e8adb19d696bc7bcfb18ccd770/drivers/virtualbox/virtualbox_windows.go#L44